zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryRequestMapper.java

```diff
@@ -129,6 +131,20 @@ public static Either<ProblemDetail, RoleQuery> toRoleQuery(final RoleSearchQuery
     return buildSearchQuery(null, sort, page, SearchQueryBuilders::roleSearchQuery);
   }
 
+  public static Either<ProblemDetail, GroupQuery> toGroupQuery(
+      final GroupSearchQueryRequest request) {
+    if (request == null) {
+      return Either.right(SearchQueryBuilders.groupSearchQuery().build());
+    }
```

Comment:
❓ So if there is no request body applied, we will just return all the groups that exist?